lib/mix/lib/mix/compilers/elixir.ex

```diff
@@ -1,7 +1,15 @@
 defmodule Mix.Compilers.Elixir do
   @moduledoc false
 
-  @manifest_vsn :v3
+  @manifest_vsn :v4
+
+  import Record
+
+  defrecordp :module, [:module, :kind, :source, :beam, :binary]
+
+  defrecordp :file, [:file, :compile, :runtime]
+
+  defrecordp :source, [:source, :files]
```

Comment:
Personally, I would put these module definitions in a single block without newlines between them. It will make the code a bit more compact and group these definitions logically. Wdyt? :)